Выполнение задания https://goo.gl/K8jJfH

unbarh opened the pull request.

unbarh committed:
Сделано задание 1

unbarh committed:
Сделано задание 2

unbarh committed:
Сделаны задания 3 и 7

unbarh committed:
Сделано задание 4

unbarh committed:
Сделаны задания 5 и 8

unbarh committed:
Сделано задание 6

unbarh committed:
Переделано задание 8

unbarh committed:
Переделано задание 1

unbarh committed:
Переделаны задания 3 и 7

unbarh committed:
Переделано задание 4

ghost commented:
При наведении курсора на кнопки курсор должен быть в виде руки. И ты не убрал outline (у кнопок он есть)

unbarh committed:
Множественные изменения:

- добавлен курсор для кнопок;
- убрано свойство outline у кнопок.

unbarh committed:
Множественные изменения

- сделан сброс свойства outline для ссылок и кнопок;
- увеличена рамка у кнопок.

ghost commented:
Принял!
